python/hail/methods/impex.py

```diff
@@ -615,16 +615,127 @@ def import_table(paths, key=[], min_partitions=None, impute=False, no_header=Fal
            reference_genome=nullable(GenomeReference),
            contig_recoding=nullable(dictof(strlike, strlike)),
            drop_chr0=bool)
-def import_plink(bed, bim, fam, min_partitions=None, delimiter='\\\\s+',
-                 missing='NA', quant_pheno=False, a2_reference=True, reference_genome=None,
-                 contig_recoding={'23': 'X', '24': 'Y', '25': 'X', '26': 'MT'}, drop_chr0=False):
+def import_plink(bed, bim, fam,
+                 min_partitions=None,
+                 delimiter='\\\\s+',
+                 missing='NA',
+                 quant_pheno=False,
+                 a2_reference=True,
+                 reference_genome=None,
+                 contig_recoding={'23': 'X',
+                                  '24': 'Y',
+                                  '25': 'X',
+                                  '26': 'MT'},
+                 drop_chr0=False):
+    """Import PLINK binary file (BED, BIM, FAM) as a :class:`.MatrixTable`.
+
+    Examples
+    --------
+
+    Import data from a PLINK binary file:
+
+    >>> ds = methods.import_plink(bed="data/test.bed",
+    ...                           bim="data/test.bim",
+    ...                           fam="data/test.fam")
+
+    Notes
+    -----
+
+    Only binary SNP-major mode files can be read into Hail. To convert your
+    file from individual-major mode to SNP-major mode, use PLINK to read in
+    your fileset and use the ``--make-bed`` option.
+
+    Hail ignores the centimorgan position (Column 3 in BIM file).
+
+    Hail uses the individual ID (column 2 in FAM file) as the sample id (`s`).
+    The individual IDs must be unique.
+
+    The resulting :class:`.MatrixTable` has the following fields:
+
+    * Row fields:
+
+        * **s** (*String*) -- Column 2 in the Fam file.
+        * **fam_id** (*String*) -- Column 1 in the FAM file. Set to missing if
+          ID equals "0".
+        * **pat_id** (*String*) -- Column 3 in the FAM file. Set to missing if
+          ID equals "0".
+        * **mat_id** (*String*) -- Column 4 in the FAM file. Set to missing if
+          ID equals "0".
+        * **is_female** (*String*) -- Column 5 in the FAM file. Set to missing
+          if value equals "-9", "0", or "N/A". Set to true if value equals "2".
+          Set to false if value equals "1".
+        * **is_case** (*String*) -- Column 6 in the FAM file. Only present if
+          ``quant_pheno`` equals False. Set to missing if value equals "-9",
+          "0", "N/A", or the value specified by ``missing``. Set to true if
+          value equals "2". Set to false if value equals "1".
+        * **quant_pheno** (*String*) -- Column 6 in the FAM file. Only present
+          if ``quantpheno`` equals True. Set to missing if value equals
+          ``missing``.
+
+    * Column fields:
+
+        * **v** (*Variant*)
```

Comment:
I said ` -- Variant (key field)`

https://ci.hail.is/repository/download/HailSourceCode_HailMainline_BuildDocs/55069:id/www/docs/stable/methods/index.html#hail.methods.balding_nichols_model